recipes/bolt/install_bolt.bat

```diff
@@ -0,0 +1,20 @@
+@echo on
+
+:: temporary prefix to be able to install files more granularly
+mkdir temp_prefix
+
+if "%PKG_NAME%" == "libbolt-devel" (
+    cmake --install ./build --prefix=./temp_prefix
+    if %ERRORLEVEL% neq 0 exit 1
+    REM only bolt libraries
+    mkdir %LIBRARY_LIB%\cmake\llvm
+    move .\temp_prefix\lib\LLVMBOLT*.lib %LIBRARY_LIB%
+    REM copy CMake metadata
+    move .\temp_prefix\lib\cmake\llvm %LIBRARY_LIB%\cmake\llvm
```

Comment:
What about include files?

Reply:
I had been wondering about that too, but there's nothing I can find. The only occurrences of `include/bolt` in the logs ([example](https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/987438/logs/33)) are during the build, not during installation. I've now done a regex-search (`include/.*bolt`) as well, and it only finds false positives

Reply:
If there's anything that goes beyond what's in `llvmdev`, it's now being installed (4f17e53d55ef07dbeefdeb356a53aa10cc86639e)

Reply:
If there are no include files, then we might as well drop `libbolt-devel` as users can't use those libraries (not easily anyways).

Reply:
I don't know sufficiently well how bolt works, but it's conceivable to me that bolt the binary needs those libraries to actually transform a binary (perhaps embedding some symbols from the support libraries along the way).

It seemed natural to split off the libraries, but if you want, I can just stuff everything into `bolt` itself.